src/modals/feedback_modal.js

```diff
@@ -0,0 +1,66 @@
+import FeedbackModalView from '../views/feedback_modal.dot'
+import ExtendedString from "../libs/string"
+import Modal from "./modal"
+
+export default class FeedbackModal {
+  constructor(results) {
+    this.modal = new Modal(this, FeedbackModalView)
+
+    this.ExtendedString = ExtendedString
+
+    this.original_results = []
+    this.displayed_results = []
+    this.current_query = ''
+
+    this.fields = {
+      result_not_found: false,
+      result_not_found_details: '',
```

Comment:
Is it expected to not have `_details` fields for all fields?

Reply:
Please check the mockup, details are only asked for a certain expected result which has not been found OR from an other chosen reason based on the next checkboxes. 
![Capture d’écran 2019-06-03 à 10 09 30](https://user-images.githubusercontent.com/1511275/58786545-1273da00-85e8-11e9-8f80-e00b315a300a.png)